README.md

```diff
@@ -20,31 +20,39 @@ This Commitizen adapter helps standardize commit messages for freeCodeCamp so
 being asked to fix incorrectly formatted commit messages will be less of a
 problem.
 
-# Local Installation
+# Installation
+
+Install as a global adapter for freeCodeCamp repos
 
 ```shell
-# Install general tool globally
-npm install -g commitizen
+# Install the commitizen cli globally
+npm install --global commitizen
 
-# Install freeCodeCamp specific adapter
-git clone https://github.com/freeCodeCamp/cz-freecodecamp.git
+# Install the freeCodeCamp commitizen adapter globally
+npm install --global cz-freecodecamp
 
-# Add freeCodeCamp plugin to be used
-cd cz-freecodecamp
-echo '{ "path": "'$(pwd)'" }' > ~/.czrc
+# Save the adapter into global config
+echo '{ "path": "cz-freecodecamp" }' > ~/.czrc
```

Comment:
I haven't been able to test this locally yet, but according to their docs, that *should* work.

Source: https://github.com/commitizen/cz-cli#conventional-commit-messages-as-a-global-utility